Kernel/Syscall.cpp

```diff
@@ -7,8 +7,11 @@
 
 #include <Kernel/API/Syscall.h>
 #include <Kernel/Arch/TrapFrame.h>
-#include <Kernel/Arch/x86/Interrupts.h>
-#include <Kernel/Memory/MemoryManager.h>
+#if ARCH(I386) || ARCH(X86_64)
+#    include <Kernel/Arch/x86/Interrupts.h>
+#elif ARCH(AARCH64)
+#    include <Kernel/Memory/MemoryManager.h>
```

Comment:
what's going on here? why do we not include MemoryManager on x86 platforms anymore?

Reply:
🫢